plover/gui_qt/suggestions_widget.py

```diff
@@ -52,6 +52,12 @@ def append(self, suggestion_list):
 
     def clear(self):
         self.suggestions.clear()
+        
+    def replace(self, suggestion_list):
+        self.clear()
+        self.append(suggestion_list)
+        scrollbar = self.suggestions.verticalScrollBar()
+        scrollbar.setValue(scrollbar.minimum())
```

Comment:
That change was made because, if the search returns enough results to overfill the suggestions window, it will automatically scroll down. This is desirable if you're using the suggestions widget to show suggestions as you type, but in the lookup window it's not, because usually the first results are the ones you want to see. Only the lookup code calls this.